lib/publishes_to_publishing_api.rb

```diff
@@ -12,10 +13,14 @@ def can_publish_to_publishing_api?
   end
 
   def publish_to_publishing_api
-    Whitehall::PublishingApi.publish_async(self)
+    run_callbacks :published do
```

Comment:
I've just tried this in dev and it also seems to cause the Stats Announcement to be pushed to search. It seems a bit odd that calling `publish_to_publishing_api` would do that.